common/exe_path.h

```diff
@@ -10,11 +10,12 @@
 namespace Carbon {
 
 // Computes the executable path for the given `argv[0]` value form `main`.
```

Comment:
```suggestion
// Computes the executable path for the given `argv[0]` value from `main`.
```
(drive-by, but I was having real trouble understanding what this meant)

Reply:
Done